src/app/components/InlineLink/index.tsx

```diff
@@ -0,0 +1,62 @@
+/** @jsx jsx */
+
+import { useContext, FC, HTMLAttributes } from 'react';
+import { jsx, Theme } from '@emotion/react';
+import Url from 'url-parse';
+
+import { FontVariant, GelFontSize } from '../../models/types/theming';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import Link from '../Link';
+import styles from './index.styles';
+
+interface Props extends HTMLAttributes<HTMLElement> {
+  allowCSR?: boolean;
```

Comment:
Can't see anything in the component standards about this but is it perhaps more readable to consistently keep optional types at the bottom of our definitions and mandatory types at the top?

Reply:
I try to arrange props alphabetically so they are easier to locate. Can you explain how it would be more readable like with what you have suggested?

Reply:
That's fine, makes sense - think it's just the way my brain has been working with TS!